src/App/Pages/Accounts/LoginPasswordlessViewModel.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Bit.App.Resources;
+using Bit.App.Utilities;
+using Bit.Core.Abstractions;
+using Bit.Core.Utilities;
+using Xamarin.CommunityToolkit.ObjectModel;
+using Xamarin.Forms;
+
+namespace Bit.App.Pages
+{
+    public class LoginPasswordlessViewModel : BaseViewModel
+    {
+        private IAuthService _authService;
+        private IPlatformUtilsService _platformUtilsService;
+        private ILogger _logger;
+        private string _logInAttempByLabel;
+        private string _deviceType;
+        private FormattedString _fingerprintPhraseFormatted;
+        private string _fingerprintPhrase;
+        private string _email;
+        private string _timeOfRequest;
+        private DateTime _requestDate;
+        private string _nearLocation;
+        private string _ipAddress;
+
+        public LoginPasswordlessViewModel()
+        {
+            _authService = ServiceContainer.Resolve<IAuthService>("authService");
+            _platformUtilsService = ServiceContainer.Resolve<IPlatformUtilsService>("platformUtilsService");
+            _logger = ServiceContainer.Resolve<ILogger>("logger");
+
+            PageTitle = AppResources.LogInRequested;
+
+            AcceptRequestCommand = new AsyncCommand(AcceptRequestAsync,
+                onException: ex => _logger.Exception(ex),
+                allowsMultipleExecutions: false);
+            RejectRequestCommand = new AsyncCommand(RejectRequestAsync,
+                onException: ex => _logger.Exception(ex),
+                allowsMultipleExecutions: false);
```

Comment:
for the `onException`, besides logging you should also inform the user that an unexpected error has occurred. Otherwise the user won't have any feedback on the UI.